services/worker-manager/src/monitor.js

```diff
@@ -171,3 +171,21 @@ MonitorManager.register({
     error: 'Error message from cloud that triggered this',
   },
 });
+
+MonitorManager.register({
+  name: 'registrationNewIntermediateCertificate',
+  title: 'Registration of New Intermediate Certificate',
+  type: 'registration-new-intermediate-certificate',
+  version: 1,
+  level: 'warning',
```

Comment:
Is this a warning because we want to eventually take some action from it?

Reply:
Thanks, I wanted some feedback on this.

One way to interpret it is "WARNING: jwhitlock's code just got called for the first time in production!". Someone may want to look at the data, download the certificate, and make sure it was correct.

It is also possible that Azure will publish a new list of expected intermediate certificates, and we should just pre-load those, and save the network request in the future. In this case, the Warning would be "a new cert was published, go look at the documentation and see if we need a PR".

In both cases, "info" might be appropriate now, or in a few months.